libs/tree-sitter-wing/grammar.js

```diff
@@ -148,29 +150,22 @@ module.exports = grammar({
         repeat(
           choice(
             $.constructor,
-            $.function_definition,
-            $.inflight_function_definition,
-            $.class_member,
-            $.inflight_class_member
+            $.method_definition,
+            $.inflight_method_definition,
+            $.class_field,
           )
         ),
         "}"
       ),
-    class_member: ($) =>
-      seq(
-        optional(field("access_modifier", $.access_modifier)),
-        optional(field("reassignable", $.reassignable)),
-        field("name", $.identifier),
-        $._type_annotation,
-        ";"
-      ),
-    inflight_class_member: ($) =>
+    class_field: ($) =>
       seq(
         optional(field("access_modifier", $.access_modifier)),
-        field("phase_modifier", $._inflight_specifier),
+        optional(field("static", $.static)),
+        optional(field("phase_modifier", $._inflight_specifier)),
```

Comment:
Does this mean we can have `static inflight` fields and methods?

(Kind of feels like we need)

Reply:
I think it should work, need to add a test.

Reply:
This works as long as the type is define inflight. See example in _static_members.w_ test. I've opened a tracking issue (#1669) for the more complex scenario, hopefully you're ok merging as is.

Reply:
I'm ok with that. Let's merge that PR in. #1669 is p1 IMO, it is similar to #1169. 
Resolving - @eladb, please unresolve or let's speak if you disagree.